plugins/cloudflare/icon_cloudflare/util/helpers.py

```diff
@@ -51,12 +51,12 @@ def clean(item_to_clean: Union[dict, list]) -> Union[dict, list]:
 
 
 def set_configuration(target: str) -> dict:
-    if validators.ipv4(target):
+    if validators.ipv4(target, cidr=False):
         return {"value": target, "target": "ip"}
-    if validators.ipv6(target):
+    if validators.ipv6(target, cidr=False):
         return {"value": target, "target": "ip6"}
```

Comment:
when was this plugin last updated? I remember these changes being needed for this validators package, but just want to double check these `cidr=False` values don't break anything, or if they were the default for these methods before.

Reply:
It was last updated a little over a year ago. The false values are only used to recognise non cidr ip addresses, as the cidr function has been removed from the validator package. I tested them out and everything works perfect! 😄